src/network-layer/default/__tests__/RelayDefaultNetworkLayer-test.js

```diff
@@ -105,9 +107,9 @@ describe('RelayDefaultNetworkLayer', () => {
     });
 
     it('sends correct data to server', () => {
-      expect(fetch).not.toBeCalled();
+      expect(fetch.mock.calls.length).toBe(0);
```

Comment:
it seems like we have a regression on jest side and `toBeCalled()` no longer works on some of the mock functions.
we're working on fixing it, but this is going to be a workaround for now